lib/common/components/ExportPatternsModal.js

```diff
@@ -0,0 +1,69 @@
+// @flow
```

Comment:
Is this file still used?

Reply:
Yep the `ExportPatternsModal` component is used.

Reply:
Where?

Reply:
In the `editorFeedSourcePanel` here: https://github.com/ibi-group/datatools-ui/pull/991/files#diff-0298c826500984d4d2e45fb8d411be5f4a20509e3803990febe3c12b7bfe7eccR67-R71

Reply:
Set up a visualization here: https://blogs.motiondevelopment.top/datatools-ui